### ISSUE TITLE
Content-Type including charset=utf-8

### ISSUE THREAD

**Issue by bermud** [opengeospatial/ets-wms11#37]
The following two tests fail when testing this endpoint
 http://www3.demis.nl/wms/wms.ashx?&ReQuEsT=GetCapabilities&SeRvIcE=WMS&VeRsIoN=1.1.1

ASP.NET adds 'charset=utf-8'.  

Maybe the test should  allow also the possibly to return ```'text/xml; charset=utf-8'.```

```
Test wms:wmsops-getfeatureinfo-params-query_layers-3 type Mandatory default result Passed (s0003/d1e1289_1)
Message d1e6196_1:
   [FAIL] Expected Content-Type 'text/xml', 
       but received 'text/xml; charset=utf-8'.
 

Test wms:wmsops-getfeatureinfo-params-query_layers-4 type Mandatory default result Passed (s0003/d1e1310_1)
Message d1e6315_1:
   [FAIL] Expected Content-Type 'text/xml', 
       but received 'text/xml; charset=utf-8'.

```

**Comment by segoerke** [comment]
This is related to https://github.com/opengeospatial/ets-wms11/issues/23 and is working with ets versions from 1.10 on (1.10 and current dev version actually)

**Comment by pgrashoff** [comment]
Yes, I also expected this was related to earlier similar erros where charset= has been added. When will this be implemented in team engine 1.9?

**Comment by segoerke** [comment]
I think this will not be fixed for the version 1.9 of the WMS tests, as 1.10 is already installed on beta. A new release is planned as far as I know soon.

**Comment by pgrashoff** [comment]
Ok, thanks Sebastian, in that case we will wait for 1.10 for our next test round :)

**Comment by bermud** [comment]
@pgrashoff, the fix (1.10) is in beta. http://cite.opengeospatial.org/te2/. Please check and let us know. 1.10 will be moved to production next week.

**Comment by pgrashoff** [comment]
Dear Luis,

 

Yes, I saw the message from Sebastian.

 

I tried the test, but it stopped at WMS_MAIN. Not sure why.

 

See the report below.

 

Kind regards,

 

Poul

 

Test wms:wms_main type Mandatory default result Passed (s0006)

 

Assertion: This WMS is valid

 

Request d1e378_1:

   Method: get

   URL: http://www3.demis.nl/wms/wms.ashx?SERVICE=WMS&REQUEST=GetCapabilities&VERSION=1.1.1

   Response from parser ::

      <WMT_MS_Capabilities version="1.1.1"><Service><Name>OGC:WMS</Name>

      <Title/>

      <Abstract>None</Abstract>

      <OnlineResource xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="http://www3.demis.nl"

                      xlink:type="simple"/>

      <ContactInformation>

         <ContactPersonPrimary>

            <ContactPerson/>

            <ContactOrganization/>

         </ContactPersonPrimary>

         <ContactElectronicMailAddress/>

      </ContactInformation>

      <Fees>none</Fees>

      <AccessConstraints>none</AccessConstraints>

   </Service>

   <Capability>

      <Request>

         <GetCapabilities>

            <Format>application/vnd.ogc.wms_xml</Format>

            <DCPType>

               <HTTP>

                  <Get><OnlineResource xmlns:xlink="http://www.w3.org/1999/xlink"

                                     xlink:href="http://www3.demis.nl/wms/wms.ashx?"

                                     xlink:type="simple"/>

                  </Get>

               </HTTP>

            </DCPType>

         </GetCapabilities>

         <GetMap>

            <Format>image/png</Format>

            <Format>image/jpeg</Format>

            <Format>image/gif</Format>

            <Format>image/bmp</Format>

            <Format>image/swf</Format>

            <DCPType>

               <HTTP>

                  <Get><OnlineResource xmlns:xlink="http://www.w3.org/1999/xlink"

                                     xlink:href="http://www3.demis.nl/wms/wms.ashx?"

                                     xlink:type="simple"/>

                  </Get>

               </HTTP>

            </DCPType>

         </GetMap>

         <GetFeatureInfo>

            <Format>text/xml</Format>

            <Format>text/plain</Format>

            <Format>text/html</Format>

            <Format>text/swf</Format>

            <Format>application/vnd.ogc.gml</Format>

            <DCPType>

               <HTTP>

                  <Get>

                     <OnlineResource xmlns:xlink="http://www.w3.org/1999/xlink"

                                     xlink:href="http://www3.demis.nl/wms/wms.ashx?"

                                     xlink:type="simple"/>

                  </Get>

               </HTTP>

            </DCPType>

         </GetFeatureInfo>

      </Request>

      <Exception>

         <Format>application/vnd.ogc.se_inimage</Format>

         <Format>application/vnd.ogc.se_blank</Format>

         <Format>application/vnd.ogc.se_xml</Format>

         <Format>text/xml</Format>

         <Format>text/plain</Format>

      </Exception>

      <VendorSpecificCapabilities/>

      <Layer noSubsets="0" opaque="0" queryable="0">

         <Title/>

         <SRS>EPSG:4326</SRS>

         <LatLonBoundingBox maxx="180" maxy="90" minx="-180" miny="-90"/>

         <BoundingBox SRS="EPSG:4326" maxx="2" maxy="6" minx="-2" miny="-1"/>

         <Layer noSubsets="0" opaque="0" queryable="1">

            <Name>cite:BasicPolygons</Name>

            <Title>cite:BasicPolygons</Title>

            <BoundingBox SRS="EPSG:4326" maxx="2" maxy="6" minx="-2" miny="-1"/>

            <Dimension name="parameter" units=""/>

            <Extent current="0" default="NR" multipleValues="0" name="parameter" nearestValue="0">ID,NR,NAME</Extent>

         </Layer>

        <Layer noSubsets="0" opaque="0" queryable="1">

            <Name>cite:Forests</Name>

            <Title>cite:Forests</Title>

            <BoundingBox SRS="EPSG:4326" maxx=".0042" maxy=".0018" minx="-.0014" miny="-.0024"/>

         </Layer>

         <Layer noSubsets="0" opaque="0" queryable="1">

            <Name>cite:Buildings</Name>

            <Title>cite:Buildings</Title>

            <BoundingBox SRS="EPSG:4326" maxx=".0024" maxy=".001" minx=".0008" miny=".0005"/>

         </Layer>

         <Layer noSubsets="0" opaque="0" queryable="1">

            <Name>cite:Lakes</Name>

            <Title>cite:Lakes</Title>

            <BoundingBox SRS="EPSG:4326" maxx=".0031" maxy="-.0001" minx=".0006" miny="-.0018"/>

         </Layer>

         <Layer noSubsets="0" opaque="0" queryable="1">

            <Name>cite:NamedPlaces</Name>

            <Title>cite:NamedPlaces</Title>

            <BoundingBox SRS="EPSG:4326" maxx=".0042" maxy=".0024" minx=".0014" miny="-.0011"/>

         </Layer>

         <Layer noSubsets="0" opaque="0" queryable="1">

            <Name>cite:Ponds</Name>

            <Title>cite:Ponds</Title>

            <BoundingBox SRS="EPSG:4326" maxx="-.0014" maxy=".002" minx="-.002" miny=".0016"/>

         </Layer>

         <Layer noSubsets="0" opaque="0" queryable="1">

            <Name>cite:Buildingcenters</Name>

            <Title>cite:Buildingcenters</Title>

            <BoundingBox SRS="EPSG:4326" maxx=".0022" maxy=".0009" minx=".001" miny=".0006"/>

         </Layer>

         <Layer noSubsets="0" opaque="0" queryable="1">

            <Name>cite:DividedRoutes</Name>

            <Title>cite:DividedRoutes</Title>

            <BoundingBox SRS="EPSG:4326" maxx="-.0026" maxy=".0024" minx="-.0032" miny="-.0024"/>

         </Layer>

         <Layer noSubsets="0" opaque="0" queryable="1">

            <Name>cite:MapNeatline</Name>

            <Title>cite:MapNeatline</Title>

            <BoundingBox SRS="EPSG:4326" maxx=".0042" maxy=".0024" minx="-.0042" miny="-.0024"/>

         </Layer>

         <Layer noSubsets="0" opaque="0" queryable="1">

            <Name>cite:RoadSegments</Name>

            <Title>cite:RoadSegments</Title>

            <BoundingBox SRS="EPSG:4326" maxx=".0042" maxy=".0024" minx="-.0042" miny="-.0024"/>

         </Layer>

         <Layer noSubsets="0" opaque="0" queryable="1">

            <Name>cite:Streams</Name>

            <Title>cite:Streams</Title>

            <BoundingBox SRS="EPSG:4326" maxx=".0036" maxy=".0024" minx="-.0004" miny="-.0024"/>

         </Layer>

        <Layer noSubsets="0" opaque="0" queryable="1">

            <Name>cite:Bridges</Name>

            <Title>cite:Bridges</Title>

            <BoundingBox SRS="EPSG:4326" maxx=".00022" maxy=".00077" minx=".00018" miny=".00063"/>

         </Layer>

      </Layer>

   </Capability>

   </WMT_MS_Capabilities>

 

Error: Unable to bind function functions:append-kvp

Result: Failed
